### PR TITLE
471 character intel

### PR DESCRIPTION
**Pull request by herpaderpaldent**
This PR requires: https://github.com/eveseat/services/pull/77 

* Introduces full server side intel view
* Introduces modal subviews

image

* removes filter/search/ordering (as it shows top interactions)

This PR resolves: https://github.com/eveseat/seat/issues/471

### CODE REVIEW

**Comment by warlof** [comment]
I have some rendering issue on the mail modal during the loading stage (seems to be an extra column which is hidden post rendering). I've also noticed there were 4 header but 5 data cells (a missing empty header for the counter or button ?)

image

Also, the counters available at the bottom of the mail modal are not updating
image

---

The note tab is missing a columnEscape for note column

image

---

Except the two upper, green for merge.

File: src/Http/Routes/Character/View.php [hunk 1]
```diff
@@ -123,20 +123,38 @@
         'uses'       => 'IntelController@getTopWalletJournalData',
     ]);
 
+    Route::get('summary/journal/details/{first_party_id}/{getJournalContent}', [
```
Comment (leonjza) [inline src/Http/Routes/Character/View.php]: `getJournalContent` should match the rest of the convention used in SeAT please. aka: `get_journal_content`

File: src/resources/views/character/intel/summary.blade.php [hunk 2]
```diff
@@ -7,11 +7,15 @@
...
 
-  <div class="panel panel-default">
-    <div class="panel-heading">
-      <h3 class="panel-title">Top Wallet Journal Interactions</h3>
-    </div>
-    <div class="panel-body">
+  <div class="nav-tabs-custom">
+    <ul class="nav nav-tabs">
+      <li class="active"><a href="#" data-toggle="tab"
+                            data-characters="single">{{trans_choice('web::seat.character',1)}}</a></li>
```
Comment (leonjza) [inline src/resources/views/character/intel/summary.blade.php]: `trans_choice('web::seat.character',1)` -> `trans_choice('web::seat.character', 1)`

File: src/Http/Routes/Character/View.php (hunk 1 above)
Comment (herpaderpaldent) [reply src/Http/Routes/Character/View.php]: that's a stupid copy-pasta error 👎 

it's obviously the method from IntelController and should be `second_party_id`

**Comment by leonjza** [comment]
LGTM

**Pull request merged**